knative-operator/pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -87,6 +87,10 @@ func (r *ReconcileKnativeServing) Reconcile(request reconcile.Request) (reconcil
 		return reconcile.Result{}, r.delete(instance)
 	}
 
+	if err := r.createConsoleCLIDownloadCR(); err != nil {
```

Comment:
I'd prefer to treat this as a "stage", even if you never use the passed instance and never return an error.